storage/client/blob/CHANGELOG.md

```diff
@@ -7,10 +7,10 @@ https://aka.ms/azure-sdk-preview1-java.
 
 **Breaking changes: New API design**
 - Operations are now scoped to a particular client:
-    - `StorageClient`: This client handles account-level operations. This includes managing service properties and listing the containers within an account.
-    - `ContainerClient`: The client handles operations for a particular container. This includes creating or deleting that container, as well as listing the blobs within that container and managing properties and metadata.
-    - `BlobClient`: The client handles operations for a particular blob. This includes creating or deleting that blob, as well as upload and download data and managing properties.
-    This BlobClient handles all blob types (block, page and append). Where operations can behave differently according to type (i.e. `upload`) the default behaviour will be block blobs unless otherwise specified.
+    - `StorageClient`: StorageURL's functionality was migrated to StorageClient. This client handles account-level operations. This includes managing service properties and listing the containers within an account.
+    - `ContainerClient`: ContainerURL's functionality was migrated to ContainerClient. The client handles operations for a particular container. This includes creating or deleting that container, as well as listing the blobs within that container.
+    - `BlobClient`: StorageURL's functionality was migrated to StorageClient. The client handles most operations, excluding upload, for an individual blob, including downloading data and working with blob properties.
```

Comment:
Shouldn't it be "BlobURL's functionality was migrated to BlobClient?

Can you also mention that TransferManager downloads were moved to BlobClient and TransferManager uploads were moved to BlockBlobClient?

Reply:
Addressed the comments.

Reply:
Line 12 should say BlobClient not StorageClient.